### PR TITLE
use constant PHP_SAPI

### PR DESCRIPTION
**Pull request by dereuromark**
Following result of https://github.com/cakephp/cakephp/pull/7100/

### CODE REVIEW

**Comment by dereuromark** [comment]
It is also consistent with Cake3 (master branch) then now.

File: lib/Cake/Console/Templates/skel/webroot/index.php [hunk 1]
```diff
@@ -77,7 +77,7 @@
 }
 
 // for built-in server
-if (php_sapi_name() === 'cli-server') {
+if (PHP_SAPI === 'cli-server') {
```
Comment (okinaka) [inline lib/Cake/Console/Templates/skel/webroot/index.php]: Is PHP_SAPI defined above?

Comment (dereuromark) [inline lib/Cake/Console/Templates/skel/webroot/index.php]: PHP native.

Comment (okinaka) [inline lib/Cake/Console/Templates/skel/webroot/index.php]: oh sorry.

**Pull request merged**